netmanager/src/views/components/DataDisplay/DeviceView/DeviceDeployStatus.js

```diff
@@ -409,41 +415,6 @@ export default function DeviceDeployStatus({ deviceData, siteOptions }) {
     return false;
   };
 
-  const updateDeviceDeploymentStatus = async (deviceId, deployStatus) => {
```

Comment:
This is no longer needed since the device status is updated from the backend on recall/deploying a device.
https://github.com/airqo-platform/AirQo-api/pull/1122

Reply:
Alright, thanks @Codebmk !